scripts/buf.gen.gogo.yaml

```diff
@@ -6,3 +6,5 @@ plugins:
       - plugins=grpc
       - Mgoogle/protobuf/any.proto=github.com/cosmos/cosmos-sdk/codec/types
       - Mcosmos/orm/v1/orm.proto=cosmossdk.io/orm
+      - Mcosmos/staking/v1beta1/tx.proto=github.com/cosmos/cosmos-sdk/x/staking/types
+      - Mcosmos/slashing/v1beta1/tx.proto=github.com/cosmos/cosmos-sdk/x/slashing/types
```

Comment:
This is to use `cosmos-sdk/x/staking/types` and `cosmos-sdk/x/slashing/types` instead of `cosmossdk.io/x/*` in generating pb.go file

Reply:
What is the difference between the two

Reply:
Without this option, the generated pb.go file imports the type of `staking` and `slashing` from `cosmossdk.io/x`, but no go package for the two module. So I replaced to import them from `cosmos/cosmos-sdk/x`.